RecoLocalCalo/HGCalRecAlgos/interface/ClusterTools.h

```diff
@@ -9,6 +9,8 @@
 #include "DataFormats/HGCRecHit/interface/HGCRecHitCollections.h"
 #include "DataFormats/CaloRecHit/interface/CaloCluster.h"
 #include "DataFormats/ParticleFlowReco/interface/HGCalMultiCluster.h"
+#include "Geometry/Records/interface/IdealGeometryRecord.h"
```

Comment:
shouldn't this instead include`CaloGeometryRecord.h`   ?
(I don't see an `IdealGeometryRecord` added in this file)

Reply:
You are right, I also hesitated a bit there. I just moved this line from the `.cc` file, but I'll try to include what you suggest.